### DISCUSSION TITLE
What is the best way to model multiple language content in edgedb?

### DISCUSSION THREAD

**Discussion by dodyg** [geldata/gel#4951] [Q&A] [answered]
In postgres, one of the common approach is to create a column per language, e.g.

country 
- name_ar
- name_en

What is a sensible way to approach this in edgedb?

**Comment by raddevon** [comment, marked as answer]
This approach looks fine to me. You might also take some inspiration from this user on our Discord: https://discord.com/channels/841451783728529451/849374705210490900/1057455954578395176

```
module default {
  scalar type Locale extending enum<en, gr>;

  required global current_locale -> Locale {
    default := Locale.en;
  };
  
  type Item {
    multi link name -> Translation {
      constraint exclusive;
    }
    link current_name := (
      select .name filter .locale = global current_locale
    );
  }

  type Translation {
    required property locale -> Locale;
    required property text -> str;
  }
}
```

There's also some interesting discussion in this message's thread along with a similar approach: https://discord.com/channels/841451783728529451/849377751370432573/1044538779576320060
